### ISSUE TITLE
Project moved to gitlab

### ISSUE THREAD

**Issue by rdemaria** [rdemaria/pytimber#80]
The project is now maintained in

https://gitlab.cern.ch/acc-co/external/pytimber-utils/

**Comment by pelson** [comment]
Just had a support request as a result of somebody following the install instructions in the README. Do you think it would perhaps be possible to archive this project (and update the README to point to the gitlab version)?

**Comment by rdemaria** [comment]
certainly, I see if there is a way to keep the data, while avoiding misusing...
